### PR TITLE
Feature/companies page

### PR DESCRIPTION
**Pull request by TimKardol**
To test:
I just manually added four company (team) rows to the database. Then build the assets.

- [x] Profile cards should have a border colour based on their sponsor tier.
- [x] Profile cards should have the default gray border if their sponsorship is either denied or not yet approved.

### CODE REVIEW

File: resources/views/livewire/auth-navigation-menu.blade.php
Comment (v-stamenova) [inline resources/views/livewire/auth-navigation-menu.blade.php:1]: In this file at the bottom there should be a hamburger menu, that needs to have the route updated as well

File: resources/views/livewire/guest-navigation-menu.blade.php
Comment (v-stamenova) [inline resources/views/livewire/guest-navigation-menu.blade.php:1]: This also has hamburger menu to be updated like the `auth-navigation-menu.blade.php` one

**Review by v-stamenova — CHANGES_REQUESTED** [review]
Except the two issues with navigation, everything works.

**Comment by v-stamenova** [comment]
If you want maybe the companies can be separated by tiers - golden at the top, separator, new row - silver, separator, new row - bronze, separator and all the others. Just throwing ideas

File: resources/views/livewire/auth-navigation-menu.blade.php
Comment (TimKardol) [reply resources/views/livewire/auth-navigation-menu.blade.php:1]: Remind me that this needs to happen for the contact page as well. Will do that in the other pull request.

File: resources/views/livewire/guest-navigation-menu.blade.php
Comment (TimKardol) [reply resources/views/livewire/guest-navigation-menu.blade.php:1]: Same comment here.

**Comment by TimKardol** [comment]
> If you want maybe the companies can be separated by tiers - golden at the top, separator, new row - silver, separator, new row - bronze, separator and all the others. Just throwing ideas

I'll keep this in mind. Might do that next week. For now I'll merge it to dev.

**Review by v-stamenova — APPROVED** [review]

**Pull request merged**